source/deploy/mattermost-changelog.md

```diff
@@ -11,6 +11,109 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 - **Cloud Releases Prior to v9.2**: See the [Mattermost Legacy Cloud Changelog](https://docs.mattermost.com/deploy/legacy-cloud-changelog.html) for details.
 ```
 
+## Release v9.8 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: 2024-05-16**
+
+### Compatibility
+ - Updated minimum required Edge and Chrome versions to 122+.
+
+```{Important}
+If you upgrade from a release earlier than v9.7, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Upgraded Nodejs to v20.11.
+ - Pre-packaged Playbooks version [v1.39.3](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v1.39.3).
+ - Pre-packaged GitLab plugin version [v1.8.1](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.1).
+ - Pre-packaged Calls version [v0.26.2](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.26.2).
+ - Combined Desktop and Mobile notifications in the user settings modal.
+ - Added a **Don't Clear** option for Do Not Disturb.
+ - Enhanced the user interface for channel introductions.
+ - Added an ephemeral message for non-team member mentions in channels.
+ - Added emoji tooltips on hover in post message.
+ - Made the appearance of several tooltips more consistent.
+ - Updated theme colors for onboarding tour points.
+ - Updated the right-hand side Thread view to use relative timestamps to be more consistent with the global Threads view.
+ - Added a total reply count to the right-hand side thread view.
+ - Added Channel Bookmarks (disabled by default).
+
+#### Administration
+ - Downloading a support packet is now extensible with plugins. If a plugin can add content to the support packet, it will be displayed in the commercial support modal. Administrators will have the option to include/exclude that from the support package.
+ - Added Channel Bookmarks permissions to the channel user role and to the channel moderation system.
+ - Added progress logs for attachments in bulk exports.
+ - Added a **System Console** option to rebuild Elasticsearch channels indexes.
+ - Obfuscated ``ReplicaLagSettings`` in the Support Packet.
+ - Improved license loading errors.
+ - Updated the keycloak docker configs and added a ``make`` command.
+ - Removed unused ``IsOAuth`` field from ``AppError``.
+ - ``bool`` is now used for ``license_is_tria``l in the Support Packet.
```

Comment:
Little typo here - i think it should be ``license_is_trial``

Reply:
Thank you!